config/locales/hyrax.en.yml

```diff
@@ -496,6 +496,7 @@ en:
         form:
           permission_update_notices:
             sharing: "The collection's sharing options have been updated."
+            participants: "The collection's sharing options have been updated."
```

Comment:
Shouldn't it be something about participants?

Reply:
@jcoyne Collections implement sharing using participants in the same way as admin sets.  But the UI/UX design wants general user facing labels to match that of works.  So collections and works both use the terminology 'sharing' in tabs and messages to the user.

The original issue with the mockup using Sharing as a label is ##1378.  @ggeisler confirmed in an earlier discussion the desire for user to have a consistent experience.